shipyard_blueprints/bigquery/shipyard_bigquery/bigquery.py

```diff
@@ -1,39 +1,225 @@
+import json
```

Comment:
I believe this code change would be picked up by the prod blueprints. You may have to move the class code changes under the shipyard folder to avoid unintentional commits when merged into main